test/emq_mgmt_SUITE.erl

```diff
@@ -64,3 +84,157 @@ t_check_app(_Config) ->
+    [[?assertMatch({match, _}, re:run(Result, Name)) || Result <- emqx_mgmt_cli:vm([Name])] || Name <- ["load", "memory", "process", "io", "ports"]],
+    [?assertMatch({match, _}, re:run(Result, "load")) || Result <- emqx_mgmt_cli:vm(["load"])],
+    [?assertMatch({match, _}, re:run(Result, "memory"))|| Result <- emqx_mgmt_cli:vm(["memory"])],
+    [?assertMatch({match, _}, re:run(Result, "process")) || Result <- emqx_mgmt_cli:vm(["process"])],
+    [?assertMatch({match, _}, re:run(Result, "io")) || Result <- emqx_mgmt_cli:vm(["io"])],
+    [?assertMatch({match, _}, re:run(Result, "ports")) || Result <- emqx_mgmt_cli:vm(["ports"])].
+
+t_trace_cmd(_) ->
+    ct:pal("start test trace"),
+    {ok, T3} = emqx_client:start_link([{host, "localhost"},
+                                       {client_id, <<"client2">>},
+                                       {username, <<"testuser2">>},
+                                       {password, <<"pass2">>}]),
+    emqx_client:connect(T3),
+    emqx_client:subscribe(T3, <<"a/b/c">>),
+    ?assertMatch({match, _}, re:run(emqx_mgmt_cli:trace(["start", "client", "client2", "log/clientid_trace.log"]), "successfully")),
+    ?assertMatch({match, _}, re:run(emqx_mgmt_cli:trace(["stop", "client", "client2"]), "successfully")),
+    ?assertMatch({match, _}, re:run(emqx_mgmt_cli:trace(["start", "client", "client2", "log/clientid_trace.log", "error"]), "successfully")),
+    ?assertMatch({match, _}, re:run(emqx_mgmt_cli:trace(["stop", "client", "client2"]), "successfully")),
+    ?assertMatch({match, _}, re:run(emqx_mgmt_cli:trace(["start", "topic", "a/b/c", "log/clientid_trace.log"]), "successfully")),
+    ?assertMatch({match, _}, re:run(emqx_mgmt_cli:trace(["stop", "topic", "a/b/c"]), "successfully")),
+    ?assertMatch({match, _}, re:run(emqx_mgmt_cli:trace(["start", "topic", "a/b/c", "log/clientid_trace.log", "error"]), "successfully")).
+
+t_router_cmd(_) ->
+    ct:pal("start test router"),
+    {ok, T3} = emqx_client:start_link([{host, "localhost"},
+                                       {client_id, <<"client2">>},
+                                       {username, <<"testuser2">>},
+                                       {password, <<"pass2">>}]),
+    emqx_client:connect(T3),
+    emqx_client:subscribe(T3, <<"a/b/c">>),
+    emqx_mgmt_cli:routes(["show", "a/b/c"]),
+    emqx_mgmt_cli:routes(["list"]),
+    ?assertMatch({match, _}, re:run(emqx_mgmt_cli:routes(["show", "a/b/c"]), "a/b/c")).
+
+t_subscriptions_cmd(_) ->
+    ct:pal("start test subscriptions cli"),
+    {ok, T4} = emqx_client:start_link([{host, "localhost"},
+                                       {client_id, <<"client4">>},
+                                       {username, <<"testuser4">>},
+                                       {password, <<"pass4">>}]),
+    emqx_client:connect(T4),
+    emqx_client:subscribe(T4, <<"/b/b/c">>),
+    ?assertMatch({match, _}, re:run(emqx_mgmt_cli:subscriptions(["add", "client4", "/b/b/c", "0"]), "ok")),
```

Comment:
`?assertEqual(["ok~n"], emqx_mgmt_cli:subscriptions(["add", "client4", "/b/b/c", "0"]))`. Is it necessary to use `regular expressions` in your code?